typings/call.ts

```diff
@@ -13,9 +13,9 @@ export interface CallHistoryItem {
   transmitterSource?: number;
   receiver: string;
   receiverSource?: number;
-  start: string;
+  start: string|number;
```

Comment:
`start` is a string as we save it as a VARCHAR for the time being. Either parseInt it somwhere else in the client/server or just keep it as a number, as we parse it in the `DailerHistory` component.
Ref: https://github.com/project-error/npwd/blob/master/phone/src/apps/dialer/components/views/DialerHistory.tsx#L95

Reply:
This was something I already talked to Taso about and he said this was the simplest solution, I can change it to a string that's what y'all want to do.